src/foam/box/SessionServerBox.java

```diff
@@ -68,22 +68,27 @@ public void send(Message msg) {
 
           if ( authenticate_ && ! auth.check(session.getContext(), "service." + spec.getName()) ) {
             Logger logger   = (Logger) x.get("logger");
-            logger.debug("missing permission", group.getId(), "service." + spec.getName());
+            logger.debug("missing permission", group != null ? group.getId() : "NO GROUP" , "service." + spec.getName());
```

Comment:
Change this to a warning so we fix them, and potentially notify on this situation.

Reply:
added the warning later, where the `group != null` is checked